.travis.yml

```diff
@@ -33,10 +32,16 @@ env:
     - MBCHARS=builtin
 
 matrix:
+  include:
+    - name: actionmailer
+      script: spec/travis-actionmailer.sh
+      rvm: 2.5
+      install: skip
+      env: BUNDLE_GEMFILE=$PWD/rails/Gemfile
```

Comment:
Is `$PWD` needed here? Can we rely on the current working directory?

Reply:
Travis sets the variable by default to $PWD/Gemfile, so I have to override it, otherwise bundle update in the rails repository will use the main Gemfile from the mail gem and it'll break the build